modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -195,6 +200,54 @@ interfaces:
       auto-dns: true
 # ...
 ----
+<1> If you are using this configuration with dnsmasq or another DNS caching system, consider using `127.0.0.1`.
+
+* Configure a status DNS search option using Dynamic Host Configuration Protocol (DHCP) and IPv6 Stateless Address AutoConfiguration (SLAAC) servers.
++
+.Static DNS search option with DHCP/autoconf name servers
+[source,yaml]
+----
+dns-resolver:
+  config:
+    search:
+      - example.com
+      - example.org
+interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+    ipv4:
+      enabled: true
+      dhcp: true
+    ipv6:
+      enabled: true
+      dhcp: true
+      autoconf: true
+# ...
+----
+
+* Configure a static DNS for a network interface with an automatic IP configuration. Note that when updating the `/etc/resolv.conf` file on a host node, you must not specify an interface (IPv4 or IPv6) in the `NodeNetworkConfigurationPolicy` (NNCP) manifest.
```

Comment:
If user are not supposed to include any interface when changing DNS in OCP, why above use cases matters?

If openshift has no intention of support `appending static DNS before auto DNS`, then including simple YAML blow is enough.

Reply:
Only included .Static DNS for network interface with automatic IP configuration